docs/resources/Stage_Instance.md

```diff
@@ -13,13 +13,13 @@ A _Stage Instance_ holds information about a live stage.
 | channel_id            | snowflake | The id of the associated Stage channel                                                                        |
 | topic                 | string    | The topic of the Stage instance (1-120 characters)                                                            |
 | privacy_level         | integer   | The [privacy level](#DOCS_RESOURCES_STAGE_INSTANCE/stage-instance-object-privacy-level) of the Stage instance |
-| discoverable_disabled | boolean   | Whether or not Stage Discovery is disabled                                                                    |
+| discoverable_disabled | boolean   | Whether or not Stage Discovery is disabled (deprecated)                                                       |
 
 ###### Privacy Level
 
 | Level      | Value | Description                                                         |
 | ---------- | ----- | ------------------------------------------------------------------- |
-| PUBLIC     | 1     | The Stage instance is visible publicly, such as on Stage Discovery. |
+| PUBLIC     | 1     | The Stage instance is visible publicly.                             |
```

Comment:
```suggestion
| PUBLIC     | 1     | The Stage instance is visible publicly. (deprecated)                |
```
Might want to add a (deprecated) here aswell since public stages dont work anymore.

Reply:
but there is still the activity iirc

Reply:
Yes it still shows in the users presence, but thats a bug, the idea of a stage being "public" was you can join it without being a member, this behavior is gone so realistically its deprecated

Reply:
ah, kk.

Reply:
@devsnek This has also been changed, this behavior has disappeared, so realistic it's outdated.